sdk/keyvault/keyvault-certificates/src/identifier.ts

```diff
@@ -39,26 +32,29 @@ export interface ParsedKeyVaultCertificatesIdentifier {
 }
 
 /**
- * Parses a KeyVaultIdentifier from a string URI.
+ * Parses the given Key Vault Certificate Id.
+ * Key Vault Ids come in the shape of URIs, for example
+ * a KeyVaultCertificate will have an `id` property, with a value similar to the following one:
+ *
+ *   https://<keyvault-name>.vault.azure.net/certificates/<certificate-name>/<unique-version-id>
+ *
+ * This URI can be parsed into segments using this function. The previous example would result in:
+ *
+ *   {
+ *      sourceId: "https://<keyvault-name>.vault.azure.net/certificates/<certificate-name>/<unique-version-id>",
+ *      vaultUrl: "https://<keyvault-name>.vault.azure.net",
+ *      version: "<unique-version-id>",
+ *      name: "<certificate-name>"
+ *   }
+ *
+ * @param {string} id The Id of the Key Vault Certificate.
```

Comment:
In TS files, we do not need specify the type for params in the docs this way

```suggestion
 * @param id The Id of the Key Vault Certificate.
```

Reply:
Thanks! I'll take that feedback for my KV-Admin PRs too.